OutOfSchool/OutOfSchool.WebApi/Services/AreaAdminService.cs

```diff
@@ -147,43 +155,22 @@ public async Task<SearchResult<AreaAdminDto>> GetByFilter(AreaAdminFilter filter
 
         filter ??= new AreaAdminFilter();
         ModelValidationHelper.ValidateOffsetFilter(filter);
-
+        var catottgs = new List<long>();
         if (currentUserService.IsMinistryAdmin())
         {
             var ministryAdmin = await ministryAdminService.GetByUserId(currentUserService.UserId);
             filter.InstitutionId = ministryAdmin.InstitutionId;
-
-            Logger.LogInformation(
-                $"Filter institutionId {filter.InstitutionId} is not equals to logined Ministry admin institutionId {ministryAdmin.InstitutionId}");
-
-            return new SearchResult<AreaAdminDto>()
-            {
-                TotalAmount = 0,
-                Entities = default,
-            };
         }
 
         if (currentUserService.IsRegionAdmin())
         {
-            var areaAdminDto = await GetByUserId(currentUserService.UserId);
-            filter.InstitutionId = areaAdminDto.InstitutionId;
-            filter.CATOTTGId = areaAdminDto.CATOTTGId;
-
-
-            if (filter.InstitutionId != areaAdminDto.InstitutionId || filter.CATOTTGId != areaAdminDto.CATOTTGId)
-            {
-                Logger.LogInformation($"Filter institutionId {filter.InstitutionId} and CATOTTGId {filter.CATOTTGId} " +
-                                      $"is not equals to logined Otg admin institutionId {areaAdminDto.InstitutionId} and CATOTTGId {areaAdminDto.CATOTTGId}");
-
-                return new SearchResult<AreaAdminDto>()
-                {
-                    TotalAmount = 0,
-                    Entities = default,
-                };
-            }
+            var regionAdminDto = await regionAdminService.GetByUserId(currentUserService.UserId);
```

Comment:
(Performance issue) As I get this project approach - almost all async methods calls are configured with `.ConfigureAwait(false)`. Please use it (at least in the code you've added)